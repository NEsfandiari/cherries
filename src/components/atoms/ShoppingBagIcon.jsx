import React, { Component } from 'react'
import styled from 'styled-components'

/** Component that renders cart icon that we have queried from contentful and passed down as props.
 */

const Container = styled.div`
  cursor: pointer;
  display: flex;
  margin-bottom: 0.2rem;
  p {
    position: relative;
    cursor: pointer;
    margin: 0;
    padding: 0;
    right: 0.8rem;
    width: 1.1rem;
    height: 1.1rem;
    color: white;
    line-height: 1.2rem;
    font-size: 0.65rem !important;
    border-radius: 1000px;
    background-color: #ff0086;
    text-align: center;
  }
`

class ShoppingBagIcon extends Component {
  render() {
    const { click } = this.props
    return (
      <Container>
        <img
          style={{
            marginRight: 20,
            marginBottom: 5,
            marginLeft: 10,
            maxWidth: 23.92,
            maxHeight: 26,
          }}
          src={this.props.cartIcon}
          alt="search-icon"
        />
      </Container>
    )
  }
}

export default ShoppingBagIcon
