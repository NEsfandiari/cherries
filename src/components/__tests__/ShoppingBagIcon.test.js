import React from 'react'
import renderer from 'react-test-renderer'
import ShoppingBagIcon from '../atoms/ShoppingBagIcon'

describe('ShoppingBagIcon', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ShoppingBagIcon />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
