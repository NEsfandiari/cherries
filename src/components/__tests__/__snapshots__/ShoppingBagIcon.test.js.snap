// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ShoppingBagIcon renders correctly 1`] = `
<div
  className="sc-bdVaJa fTFgr"
>
  <img
    alt="search-icon"
    style={
      Object {
        "marginBottom": 5,
        "marginLeft": 10,
        "marginRight": 20,
        "maxHeight": 26,
        "maxWidth": 23.92,
      }
    }
  />
</div>
`;
